### PR TITLE
Added memory option for --key-value-storage

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added memory option for --key-value-storage

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -511,6 +511,8 @@ If a name is not specified, the name section is not included in the client ID. F
 The key-value storage to use. Use this option only if using a storage system provided with a
 plugin. The default is `rocksdb`.
 
+For ease of use when using a debugger, use the `memory` option.
```
Comment (NicolasMassart) [inline docs/Reference/CLI/CLI-Syntax.md:514]: Add a link to the option?

Comment (shemnon) [reply docs/Reference/CLI/CLI-Syntax.md:514]: https://github.com/hyperledger/besu/pull/1617 ?

I think this needs more caveats that tell the casual reader "You probably don't want this option".  The release notes say "Added `memory` as an option to `--key-value-storage`.  This ephemeral storage is intended for sync testing and debugging. "

Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md:514]: Updated to make this clearer

Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md]: @NicolasMassart - I don't understand your comment?

**Comment by MadelineMurray** [comment]
@shemnon - is the updated version better?

**Review by NicolasMassart — APPROVED** [review]

**Review by shemnon — APPROVED** [review]

**Pull request merged**